data/modules/Equipment/Weapons.lua

```diff
@@ -0,0 +1,354 @@
+	provides_slots = {
+		Slot:clone { id = "1", type = "missile", size = 2, hardpoint = true },
+		Slot:clone { id = "2", type = "missile", size = 2, hardpoint = true },
+	},
+	icon_name="equip_missile_unguided"
+})
+
+Equipment.Register("missile_rack.341", EquipType.New {
+	l10n_key="MISSILE_RACK_341",
+	price=150, purchasable=true, tech_level=1,
+	volume=0.0, mass=0.5,
+	slot = { type = "pylon.rack", size=3, hardpoint=true },
+	provides_slots = {
+		Slot:clone { id = "1", type = "missile", size = 1, hardpoint = true },
+		Slot:clone { id = "2", type = "missile", size = 1, hardpoint = true },
+		Slot:clone { id = "3", type = "missile", size = 1, hardpoint = true },
+		Slot:clone { id = "4", type = "missile", size = 1, hardpoint = true },
+	},
+	icon_name="equip_missile_unguided"
+})
+
+Equipment.Register("missile_rack.212", EquipType.New {
+	l10n_key="MISSILE_RAIL_S2",
+	price=150, purchasable=true, tech_level=1,
+	volume=0.0, mass=0.1,
+	slot = { type = "pylon.rack", size=2, hardpoint=true },
+	provides_slots = {
+		Slot:clone { id = "1", type = "missile", size = 2, hardpoint = true },
+	},
+	icon_name="equip_missile_unguided"
+})
+
+Equipment.Register("missile_rack.221", EquipType.New {
+	l10n_key="MISSILE_RACK_221",
+	price=150, purchasable=true, tech_level=1,
+	volume=0.0, mass=0.2,
+	slot = { type = "pylon.rack", size=2, hardpoint=true },
+	provides_slots = {
+		Slot:clone { id = "1", type = "missile", size = 1, hardpoint = true },
+		Slot:clone { id = "2", type = "missile", size = 1, hardpoint = true },
+	},
+	icon_name="equip_missile_unguided"
+})
+
+Equipment.Register("missile_rack.111", EquipType.New {
+	l10n_key="MISSILE_RAIL_S1",
+	price=150, purchasable=true, tech_level=1,
+	volume=0.0, mass=0.1,
+	slot = { type = "pylon.rack", size=1, hardpoint=true },
+	provides_slots = {
+		Slot:clone { id = "1", type = "missile", size = 1, hardpoint = true },
+	},
+	icon_name="equip_missile_unguided"
+})
+
+--===============================================
+-- Internal Missile Bays
+--===============================================
+
+Equipment.Register("missile_bay.opli_internal_s2", EquipType.New {
+	l10n_key="OPLI_INTERNAL_MISSILE_RACK_S2",
+	price=150, purchasable=true, tech_level=1,
+	volume=5.0, mass=0.5,
+	slot = { type = "missile_bay.opli_internal", size=2, hardpoint=true },
+	provides_slots = {
+		Slot:clone { id = "1", type = "missile", size = 2, hardpoint = true },
+		Slot:clone { id = "2", type = "missile", size = 2, hardpoint = true },
+		Slot:clone { id = "3", type = "missile", size = 2, hardpoint = true },
+		Slot:clone { id = "4", type = "missile", size = 2, hardpoint = true },
+		Slot:clone { id = "5", type = "missile", size = 2, hardpoint = true },
+	},
+	icon_name="equip_missile_unguided"
+})
+
+Equipment.Register("missile_bay.bowfin", EquipType.New {
+	l10n_key="OKB_KALURI_BOWFIN_MISSILE_RACK",
+	price=150, purchasable=true, tech_level=1,
+	volume=0.0, mass=0.2,
+	slot = { type = "missile_bay.bowfin", size=2, hardpoint=true },
```

Comment:
```suggestion
Equipment.Register("missile_bay.bowfin_internal", EquipType.New {
	l10n_key="OKB_KALURI_BOWFIN_MISSILE_RACK",
	price=150, purchasable=true, tech_level=1,
	volume=0.0, mass=0.2,
	slot = { type = "missile_bay.bowfin_internal", size=2, hardpoint=true },
```

Alternatively the "bowfin.json" ship configuration must be modified `missile_bay.bowfin_internal` -> `missile_bay.bowfin`